src/Abp/Application/Navigation/MenuItemDefinition.cs

```diff
@@ -108,7 +117,7 @@ public class MenuItemDefinition : IHasMenuItemDefinitions
             Icon = icon;
             Url = url;
             RequiresAuthentication = requiresAuthentication;
-            RequiredPermissionName = requiredPermissionName;
```

Comment:
Need to restore this and [this](https://github.com/aspnetboilerplate/aspnetboilerplate/pull/2289/files#diff-e101fb6f09f5d4ac41263b9b35b3125bL95)